src/code/z_scene_table.c

```diff
@@ -810,128 +810,32 @@ EntranceInfo gEntranceTable[] = {
     ENTRANCE(0x5C, 0x08, 0, 1, 0x03, 0x03), ENTRANCE(0x5C, 0x08, 0, 1, 0x03, 0x03),
 };
 
-#define TITLED_SCENE(name, title, unk_10, config, unk_12)                                            \
-    {                                                                                                \
-        (u32) _##name##SegmentRomStart, (u32)_##name##SegmentRomEnd, (u32)_##title##SegmentRomStart, \
-            (u32)_##title##SegmentRomEnd, unk_10, config, unk_12, 0                                  \
-    }
+// Linker symbol declarations (used in the table below)
+#define DEFINE_SCENE(name, title, _2, _3, _4, _5) \
+    DECLARE_ROM_SEGMENT(name)                     \
+    DECLARE_ROM_SEGMENT(title)
+
+#include "tables/scene_table.h"
+
+#undef DEFINE_SCENE
 
-#define UNTITLED_SCENE(name, unk_10, config, unk_12) \
-    { (u32) _##name##SegmentRomStart, (u32)_##name##SegmentRomEnd, 0, 0, unk_10, config, unk_12, 0 }
+// Scene Table definition
+#define DEFINE_SCENE(name, title, _2, config, unk_10, unk_12) \
+    { ROM_FILE(name), ROM_FILE(title), unk_10, config, unk_12, 0 },
+
+// Handle `none` as a special case for scenes without a title card
+#define _noneSegmentRomStart 0
+#define _noneSegmentRomEnd 0
```

Comment:
```suggestion
#define _noneSegmentRomStart NULL
#define _noneSegmentRomEnd NULL
```
It seems slightly more "correct" to define these to null since it's a stand-in for pointer-typed segment symbols?

Reply:
#1089 changes vromStart and vromEnd to `uintptr_t`: https://github.com/zeldaret/oot/blob/b9c0124de42099a5b9c614155badc96ee311a949/include/z64scene.h#L6-L9
so yes, if we do see rom/vrom offsets as pointers (I guess we are forced to by using symbols)